packages/testlab/test/integration/test-sandbox.integration.ts

```diff
@@ -93,13 +103,14 @@ describe('TestSandbox integration tests', () => {
     await sandbox.delete();
   }
 
-  async function compareFiles(path1: string) {
-    const file = await readFile(path1, 'utf8');
-    expect(file).to.equal(fileContent);
+  async function compareFiles(original: string, copied: string) {
```

Comment:
Nitpick: this function name should have the word `expect` in it so that it makes it clear that when the function is called it's doing an assertion. Something like `expectFilesToBeIdentical`